tests/Paramore.Brighter.DynamoDB.Tests/Outbox/When_there_are_dispatched_messages_in_the_outbox.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Paramore.Brighter.Outbox.DynamoDB;
+using Xunit;
+
+namespace Paramore.Brighter.DynamoDB.Tests.Outbox;
+
+[Trait("Category", "DynamoDB")]
+public class DynamoDbOutboxDispatchedMessageTests : DynamoDBOutboxBaseTest
+{
+    private readonly Message _message;
+    private readonly DynamoDbOutbox _dynamoDbOutbox;
+
+    public DynamoDbOutboxDispatchedMessageTests()
+    {
+        _message = CreateMessage("test_topic");
+        _dynamoDbOutbox = new DynamoDbOutbox(Client, new DynamoDbConfiguration(OutboxTableName));
+    }
+
+    [Fact]
+    public async Task When_there_are_dispatched_messages_in_the_outbox_async()
+    {
+        var context = new RequestContext();
+        await _dynamoDbOutbox.AddAsync(_message, context);
+        await _dynamoDbOutbox.MarkDispatchedAsync(_message.Id, context);
+
+        await Task.Delay(1000);
```

Comment:
It's possible we could use FakeTimeProvider to advance a time/date since marked as dispatched and allow us to avoid having to delay here